kubernetes/linux/main.sh

```diff
@@ -841,8 +845,6 @@ else
 fi
 source ~/.bashrc
 
-dpkg -l | grep mdsd | awk '{print $2 " " $3}'
```

Comment:
do we have a replace command for extracting the mdsd info?

Reply:
Have added all these as part of the package_versions.txt